rwgame/RWGame.cpp

```diff
@@ -400,45 +408,37 @@ int RWGame::run() {
         }
         RW_PROFILE_END();
 
-        auto now = clock.now();
-        float timer =
-            std::chrono::duration<float>(now - last_clock_time).count();
-        last_clock_time = now;
-        accum += timer * timescale;
+        auto now = chrono::steady_clock::now();
+        auto deltaTime = chrono::duration<float>(now - lastFrame).count();
+        lastFrame = now;
+        accumulatedTime += deltaTime;
+
+        if(!world->isPaused()) {
+            world->dynamicsWorld->stepSimulation(deltaTime * timescale, kPhysicsTimeStep, kMaxPhysicsSubSteps);
```

Comment:
I got the arguments the wrong way around here, the kPhysicsTimeStep argument should be last.